src/gateway/reference/configuration.md

```diff
@@ -3595,6 +3595,24 @@ Defines the names of the Vault v2 KV path at which symmetric keys are found.
 
 
 
+### keyring_vault_auth_method
```

Comment:
if you haven't done it already, this doc is generated by editing this file in `kong-ee` https://github.com/Kong/kong-ee/blob/master/kong.conf.default

Reply:
I have now done that.